packages/components/input/src/input.vue

```diff
@@ -297,6 +303,17 @@ export default defineComponent({
       calcIconOffset('suffix')
     }
 
+    const setHtmlMaxLength = (text: string) => {
+      const codeUnitsLength = text.length
+      const textLength = toArray(text).length
+      htmlMaxLength.value =
+        !!attrs.value.maxlength &&
```

Comment:
Can we use `isNumber` here for the assertion? For instance:

```ts
import { isNumber } from '@element-plus/utils`

htmlMaxLength.value = isNumber(attrs.value.maxlength) && restConditions
```

Reply:
I'm for it. I have already fixed.

Reply:
Resolved in https://github.com/element-plus/element-plus/pull/6130/commits/2ad48d4615a9bdacfb91fcb782ce332b383365a3#diff-740318b89e9e14e0ecf7296023cfe04687fea7371612db4f9b5d456d1648b2a7R313